### DISCUSSION TITLE
About Page

### DISCUSSION THREAD

**Discussion by WhyAydan** [transitive-bullshit/nextjs-notion-starter-kit#381] [General]
Hey,

How did you get your about page to have the fluid bit at the top?

Aydan

**Comment by 00f2ff** [comment]
Looks like WebGL or Three.js. Same layout as the cover photos. There are probably react libs to help with embedding

**Comment by transitive-bullshit** [comment]
You can see the full changes from the main branch by visiting the `transitive-bullshit` branch.

Specifically, check out https://github.com/transitive-bullshit/nextjs-notion-starter-kit/blob/transitive-bullshit/components/HeroHeader.tsx and https://github.com/transitive-bullshit/nextjs-notion-starter-kit/blob/transitive-bullshit/components/NotionPage.tsx#L232
